DolbyIO/DolbyIO.uplugin

```diff
@@ -10,24 +10,12 @@
 	"DocsURL": "https://github.com/DolbyIO/comms-sdk-unreal",
 	"MarketplaceURL": "com.epicgames.launcher://ue/marketplace/product/OFFERID",
 	"SupportURL": "https://github.com/DolbyIO/comms-sdk-unreal",
-	"CanContainContent": true,
+	"CanContainContent": false,
 	"IsBetaVersion": true,
 	"Installed": false,
 	"Modules": [
 		{
-			"Name": "DolbyIoConferenceModule",
-			"Type": "Runtime",
-			"LoadingPhase": "Default",
-			"WhitelistPlatforms": [ "Win64", "Mac" ]
-		},
-		{
-			"Name": "SdkAccessModule",
-			"Type": "Runtime",
-			"LoadingPhase": "Default",
-			"WhitelistPlatforms": [ "Win64", "Mac" ]
-		},
-		{
-			"Name": "DolbyIoSampleConferenceModule",
+			"Name": "DolbyIOModule",
 			"Type": "Runtime",
 			"LoadingPhase": "Default",
 			"WhitelistPlatforms": [ "Win64", "Mac" ]
```

Comment:
is Mac really in the whitelist?

Reply:
This just means the plugin is supported on Mac as well, which is still the case until we're told otherwise :)